src/components/content/home/isv/ServicesDashboardByIsv.tsx

```diff
@@ -0,0 +1,117 @@
+import ServicesDashboardByIsvError from './ServicesDashboardByIsvError';
+import { ServicesDashboardByIsvSkeleton } from './ServiceDashboardByIsvSkeleton';
+import useListRegisteredServicesQuery from './useListRegisteredServicesQuery';
+
+export function ServicesDashboardByIsv(): React.JSX.Element {
+    const listDeployedServicesByIsvQuery = useListDeployedServicesByIsvQuery();
+    const listRegisteredServicesByIsvQuery = useListRegisteredServicesQuery();
+    let successfulDeploymentsCount: number = 0;
+    let failedDeploymentsCount: number = 0;
+    let successfulDestroysCount: number = 0;
+    let failedDestroysCount: number = 0;
+    let registeredServicesCount: number = 0;
+
+    if (listDeployedServicesByIsvQuery.data !== undefined && listDeployedServicesByIsvQuery.data.length > 0) {
+        listDeployedServicesByIsvQuery.data.forEach((serviceItem: ServiceVo) => {
+            if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_SUCCESSFUL) {
+                successfulDeploymentsCount++;
+            }
+            if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DEPLOYMENT_FAILED) {
+                failedDeploymentsCount++;
+            }
+            if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DESTROY_SUCCESSFUL) {
+                successfulDestroysCount++;
+            }
+            if (serviceItem.serviceDeploymentState === ServiceVo.serviceDeploymentState.DESTROY_FAILED) {
+                failedDestroysCount++;
+            }
+        });
+    }
+    if (listRegisteredServicesByIsvQuery.data !== undefined && listRegisteredServicesByIsvQuery.data.length > 0) {
+        registeredServicesCount = listRegisteredServicesByIsvQuery.data.length;
+    }
+
+    if (listDeployedServicesByIsvQuery.isError || listRegisteredServicesByIsvQuery.isError) {
+        const errorToDisplay = listDeployedServicesByIsvQuery.isError
+            ? listDeployedServicesByIsvQuery.error
+            : listRegisteredServicesByIsvQuery.error;
+
+        return <ServicesDashboardByIsvError error={errorToDisplay} />;
+    }
+
+    if (
+        listDeployedServicesByIsvQuery.isPending ||
+        listDeployedServicesByIsvQuery.isFetching ||
+        listRegisteredServicesByIsvQuery.isPending ||
+        listRegisteredServicesByIsvQuery.isFetching
+    ) {
+        return <ServicesDashboardByIsvSkeleton />;
+    }
+
+    return (
+        <>
+            <Card title='Services Dashboard' bordered={true}>
+                <Row gutter={16} justify='start'>
+                    <Col span={12} className={'dashboard-container-class'}>
+                        <Statistic
+                            title='Services Registered'
+                            value={registeredServicesCount}
+                            loading={listDeployedServicesByIsvQuery.isLoading}
+                            valueStyle={{ color: '#3f8600', textAlign: 'center' }}
```

Comment:
I had also requested to keep the alignment same for the end user dashboard. can you please check that as well?